### DISCUSSION TITLE
Aircraft types support

### DISCUSSION THREAD

**Discussion by rsirac** [junzis/openap#25] [Q&A]
Hi @junzis!

I am part of the Thales team using this project. 

In our latest tests, we try to compute fuelflow on an aircraft B763 which seems to be in the available list in the directory **data/aircraft** but it is not present in the directory **data/dragpolar** which raises an exception at runtime : 
_RuntimeError: B763 drag polar not avaiable._

By the way, there is a little typing error on the word "available"

Do you think it is possible to add the support for this aircraft type?

Thank you!

Best Regards

**Comment by junzis** [comment]
Hi @Rsirac, Indeed, for the moment, I don't have enough B767 data to construct the drag polar, since there are not many flying around. I will try to see if I can get some good trajectories from OpenSky. Stay tuned.

Thank you also for spotting the typo. I will fix that in the next commit.
